### ISSUE TITLE
make Docs instructions in Contributing instructions does not work on Windows operating system.

### ISSUE THREAD

**Issue by rwkozar** [napari/napari#4828]
## 📚 Documentation
In contributing docs, the Building the docs section includes an option to use "make Docs" to generate documentation.  This does not work properly in Windows.

**Comment by DragaDoncila** [comment]
Thanks for raising this @rwkozar! I want to expand a bit and note that some of the other commands we use in our contributor's guide e.g. `rm -rf` don't work as expected on Windows. I think it's really important for us to provide these instructions, or show how to set up WSL for windows, or at the very least mention upfront that some of the instructions are windows specific.

**Comment by Czaki** [comment]
@rwkozar What did you mean that it is not working correctly? Do you meet this error? Or something different? 

```
Extension error (napari_sphinx_theme):
Handler <function add_google_calendar_secrets at 0x0000023702186940> for event 'build-finished' threw an exception (exception: 'charmap' codec can't decode byte 0x81 in position 206595: character maps to <undefined>)
make: *** [Makefile:9: docs] Error 2
```

**Comment by DragaDoncila** [comment]
@Czaki `make` is not pre-installed on windows so the error is just a generic "no valid command"

**Comment by Czaki** [comment]
python is also not pre-installed on Windows. Make could also not be installed on Ubuntu. 

But as you could see in my previous comment and in PR to `napar-sphix-theme` building docs do not work on Windows because of the bug in `napari-sphinx-theme`. So my question is if @rwkozar meet the same problem as me or reports a lack of documentation that there is a need to install make to do this step. 

googling how to install make is a smaller problem. I can not check at this moment but it looks like anaconda ships with `make`.

**Comment by DragaDoncila** [comment]
> So my question is if @rwkozar meet the same problem as me or reports a lack of documentation that there is a need to install make to do this step.

@Czaki ah I see, I wasn't aware of that existing issue. I should mention I answered the question with such authority as I am currently with @rwkozar at SciPy sprints, so I know the error he is referring to. I therefore asked him to raise this issue as many of the contributor's docs are not suitable for Windows users, even outside of the `make docs` command.

**Comment by Czaki** [comment]
Alternative for windows, if the user does not have `make`, is to execute `sphinx-build -b html . ./_build` from `docs` directory, but until https://github.com/napari/napari-sphinx-theme/pull/119 is merged and released, the building docs on windows will fail. 

other problems could be solved by using WSL or just bash for windows (for example, from git for windows https://git-scm.com/download/win)

**Comment by DragaDoncila** [comment]
@Czaki yep, I totally agree - perhaps we can edit this issue title to reflect the broader scope i.e. improving contributor's guide instructions for windows users. This would include suggestions to use WSL or bash for windows
